tests/Aspire.Hosting.Containers.Tests/FromDockerfileTests.cs

```diff
@@ -790,6 +787,30 @@ ARG MESSAGE
         COPY --from=builder /app/static/aspire.html /app/static
         RUN --mount=type=secret,id=FILE_SECRET cp /run/secrets/FILE_SECRET /app/static/FILE_SECRET.txt
         RUN --mount=type=secret,id=ENV_SECRET cp /run/secrets/ENV_SECRET /app/static/ENV_SECRET.txt
+        RUN sleep 40
         """;
 }
 
+internal static class RetryExtensions
+{
+    private static ResiliencePipeline? s_pipeline;
```

Comment:
static?

Reply:
Yeah, my understanding is that these are reusable.